app/pages/project/affinity/AntiAffinityGroupPage.tsx

```diff
@@ -172,11 +202,25 @@ export default function AntiAffinityPage() {
         <CardBlock.Header
           title="Members"
           description="Instances in this anti-affinity group"
-        />
+        >
+          <CreateButton
```

Comment:
Let's use a regular button without the add roundel. We're not currently using it inline in these card blocks.